File: Java/02- Relacionando Classes/16- Contrato/Cliente.java

package com.mycompany.mavenproject6;

public class Cliente extends Fisica{
    private String protocoloAtendimento;

    public Cliente(String protocoloAtendimento, Sexo sexo, EstadoCivil estadocivil, String dataNascimento, String nome, String telefone, String email, Endereco endereco) {
        super(sexo, estadocivil, dataNascimento, nome, telefone, email, endereco);
        this.protocoloAtendimento = protocoloAtendimento;
    }

    


    public String getProtocoloAtendimento() {
        return protocoloAtendimento;
    }

    public void setProtocoloAtendimento(String protocoloAtendimento) {
        this.protocoloAtendimento = protocoloAtendimento;
    }

    @Override
    public String toString() {
        return super.toString() +
                "\n=====Cliente=====" +                
                "\nProtocolo de Atendimento: " + protocoloAtendimento;
    }
    
    
}

File: Python/04- POO/03- Atividades Pontuadas/Atividade 01/models/cliente.py
from models.fisica import Fisica
from models.endereco import Endereco
from models.enum.sexo import Sexo

class Cliente(Fisica):
    def __init__(self, nome: str, telefone: str, email: str, endereco: Endereco, cpf: str, rg: str, data_nascimento: str, sexo: Sexo, protocolo_atendimento: int):
        super().__init__(nome, telefone, email, endereco, cpf, rg, data_nascimento, sexo)
        self.protocolo_atendimento = protocolo_atendimento

    def __str__(self) -> str:
        return (f"{super().__str__()}"
                f"\nCliente: {self.nome}"
                f"\nProtocolo: {self.protocolo_atendimento}"
               )